src/components/CopyButton/CopyButton.js

```diff
@@ -99,8 +99,9 @@ export default class CopyButton extends Component {
           <Copy16
             className={`${prefix}--snippet__icon`}
             aria-label={iconDescription}
-            alt={iconDescription}
-          />
+            alt={iconDescription}>
```

Comment:
is `alt` a valid attribute here? I think the `<title>` element is supposed to replace our invalid usage of `alt` attribute on SVGs

Reply:
Great catch @emyarod! - Fixed.